Cargo.toml

```diff
@@ -11,6 +11,7 @@ client = ["dep:reqwest", "dep:url"]
 server = ["dep:actix-cors", "dep:actix-web"]
 
 [dependencies]
+regex = "1.10.0"
```

Comment:
Sorry, I forgot to mention this on the last PR. Can we move this line below `getrandom`, and include a blank line? Please move `percentage` too.

While there are no hard-and-fast rules about `Cargo.toml` formatting, I'm trying to keep things roughly grouped together, and this first group is for `egui` and related dependencies.

Reply:
https://github.com/StanfordLegion/prof-viewer/pull/32/commits/230f9ba520f29253eb7437fee464ca969b74337a